samples/attested_tls/AttestedTLSREADME.md

```diff
@@ -31,20 +31,24 @@ combination of HW and SW gaining the trust of a remote provider or producer.
 
   By taking advantage of the fact that TLS involving parties use public-key cryptography for identity authentication during the [TLS handshaking process](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_handshake), the Attested TLS feature uses a self-signed X509.V3 certificate to represent a TLS endpoint's identity. We make this certificate cryptographically bound to this specific enclave instance by adding a custom certificate extension (called quote extension) with this enclave's attestation quote that has the certificate's public key information embedded.
 
-  A new API oe_get_attestation_certificate_with_evidence was added for generating such a self-signed certificate for use in the TLS handshaking process
+  There was an API `oe_get_attestation_certificate_with_evidence()` for generating such a self-signed certificate for use in the TLS handshaking process. However, a new API `oe_get_attestation_certificate_with_evidence_v2()` was added to replace the original function so that a list of optional user-defined format-specific parameters can be passed during the certificate-generating process.
```

Comment:
I don't think it's necessary to state both `oe_get_attestation_certificate_with_evidence()` and `oe_get_attestation_certificate_with_evidence_v2()`. Just say "Call `oe_get_attestation_certificate_with_evidence_v2()` to generate...". And it's not new anymore. No need to say "A new API...".